paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -331,12 +332,15 @@ async def get_http_utilization_for_all_tasks(
     utilization = []
     service = marathon_service_config.get_service()
 
-    futures = [
-        asyncio.ensure_future(
-            get_http_utilization_for_a_task(task, service, endpoint, json_mapper)
-        )
-        for task in marathon_tasks
-    ]
+    async with aiohttp.ClientSession() as session:
```

Comment:
can you leave a comment behind of why we do it like this, rather than one session per request? I've got a feeling we'll end up coming back to this one day and pick it as an easy place to make a performance improvement

Reply:
added